geometry_msgs/Vector3[] head_vectors
